maintenance_evolution.Rmd

```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
 
 Once a function is made defunct, its tests are simply removed.
 
+## Archiving packages {#archivalguidance}
+
+Software generally has a finite lifespan, and packages may eventually need to be archived. Archived packages are moved to a dedicated GitHub organization, [ropensci-archive](https://github.com/ropensci-archive). Prior to archiving, the contents of the README file should be moved to an alternative location (such as "README-OLD.md"), and replaced with minimal contents including the following:
+
+- Title with name of package
+- A repostatus.org badge of "unsupported" for formerly released packages, or "abandoned" for former concept or WIP packages.
```

Comment:
Could we add the Markdown blocks to copy-paste here?

Reply:
I did have that, but decided text was neater. But i've just pushed a commit which reverts back to template.